scripts/build.py

```diff
@@ -14,7 +14,7 @@ def main(base_dir):
 
 
 def get_release_version():
-    version = f'{os.environ["RELEASE_VERSION"]}'
```

Comment:
giving a default version may cause releasing to pip and npm to load the wrong version?